pkg/querier/querier.go

```diff
@@ -41,6 +42,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.BatchIterators, "querier.batch-iterators", false, "Use batch iterators to execute query, as opposed to fully materialising the series in memory.  Takes precedent over the -querier.iterators flag.")
 	f.BoolVar(&cfg.IngesterStreaming, "querier.ingester-streaming", false, "Use streaming RPCs to query ingester.")
 	f.IntVar(&cfg.MaxSamples, "querier.max-samples", 50e6, "Maximum number of samples a single query can load into memory.")
+	f.DurationVar(&cfg.IngesterMaxQueryLookback, "querier.query-ingesters-within", 24*time.Hour, "Maximum lookback beyond which queries are not sent to ingester.")
```

Comment:
Hmm, minor nit. Should this be `0`? Making this `24h` could break deployments that use like `48h` chunk time :P 

This is an esoteric, but the point being this modifies current behaviour. Am I being too pendantic?

/cc @bboreham @tomwilkie

Reply:
I don't know anyone who uses chunks that big...

But yeah, we can disable this by default.